dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/oracle/task/ResultMessageTask.java

```diff
@@ -49,6 +50,11 @@ public static NoResultTask create(QueryGroup group, List<StatsJdbcTask> tasks) {
     return new ResultMessageTask(condition, group);
   }
 
+  public static NoResultTask createNoFilter(QueryGroup group, List<Task<?>> tasks) {
```

Comment:
I try to understand the naming (in particular `NoFilter`) in the context of the `create` method above. It seems to me that the filtering (applied in the `create` method above) makes it a bit difficult to understand. I would suggest separating the filtering to another method and have just one `create` method.

Before:
```
ResultMessageTask.create(queryGroup, tasks);
```

After:
```
ResultMessageTask.create(queryGroup, queryGroup.filter(tasks))
```

Reply:
Done. I've taken another look at this and it turns out that `create` and `createNoFilter` do the same thing in two ways. The version with the filter looks hackier to me - it expects a combined list of all tasks, even though we know that the tasks it will extract must have been created separately to begin with.

I've removed the filter version. Since there's only one method, it can be just called `create`.